tsdb/series_file.go

```diff
@@ -900,3 +746,5 @@ func hashReader(r io.Reader) ([]byte, error) {
 	}
 	return h.Sum(nil), nil
 }
+
+func hexdump(data []byte) { os.Stderr.Write([]byte(hex.Dump(data))) }
```

Comment:
This isn't called by anything; maybe define it in a test file if it's still useful?

Reply:
Removed in 3900c94.